apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js

```diff
@@ -18,7 +18,7 @@ const InviteNewOrganizationEmployeeService = new GQLCustomSchema('InviteNewOrgan
     types: [
         {
             access: true,
-            type: 'input InviteNewOrganizationEmployeeInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, email: String, phone: String!, name: String, role: OrganizationEmployeeWhereUniqueInput, position: String, specializations: [TicketCategoryClassifierWhereInput!]}',
+            type: 'input InviteNewOrganizationEmployeeInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, email: String, phone: String!, name: String, role: OrganizationEmployeeWhereUniqueInput!, position: String, specializations: [TicketCategoryClassifierWhereInput!]}',
```

Comment:
Why role has OrganizationEmployeeWhereUniqueInput and not OrganizationEmployeeRoleWhereUniqueInput?

Reply:
fixed!